docs/content/node-operation/full-observer-node.mdx

```diff
@@ -1,38 +1,221 @@
 ---
-title: Full Observer Node
+title: Observer Node
 sidebar_title: Observer Node
-description: Running your own full observer node
+description: Running an observer node locally
 ---
 
-<Callout type="warning">
-  This is an alpha release and is known to contain bugs and is missing functionality. We are currently changing this codebase to address these issues. In the meantime please avoid building a dependency on this code until it's re-released to open beta.
-</Callout>
+An observer node is similar to an access node and provides a locally accessible, continuously updated and verified copy of the block data. It serves the Access gRPC API but, unlike an access node, an observer node does not need to be staked and **anyone** can run it without being added to the approved list of nodes.
 
-A full observer node is an unstaked access node that follows the flow consensus
-and serves the Access API using local data. This allows anyone to setup an access
-node for local queries without the need for staking.
+The observer node bootstraps by connecting to an access node and becoming part of the public network comprised of access nodes and other observer nodes. It then continuously receives blocks that are being added to the chain either directly from the access node or from other observer nodes that are part of the public network.
+However, it makes no trust assumption of the upstream access node or the observer node which is providing the block and locally verifies that the blocks that are received are the correct extension of the chain e.g After receiving valid blocks A, B and C when it receives block D, it verifies that block D is indeed signed by the consensus nodes and is a valid next block. The block data that is received is indexed and made available via the Access gRPC API. For Collection, Transactions and Account queries, it delegates those requests to the upstream access node. Similarly, transactions and scripts sent to an observer node are also forwarded to the upstream access node. Future versions of the observer node will be able to serve this data locally as well.
```

Comment:
> makes no trust assumption of the upstream access node

Observers still must get the genesis block from a trusted source. This statement is true if the observer uses the `root-protocol-state-snapshot.json` file from the beginning of the spork. If it uses an AN to get its current snapshot, it must trust that AN.